packages/amplify-migration-tests/src/__tests__/update_tests/function_migration_update.test.ts

```diff
@@ -116,6 +121,7 @@ describe('amplify function migration', () => {
       runtimes: [runtime],
     };
 
+    // toggle to latest
```

Comment:
Does `// toggle to latest` refer to the `testingWithLatestCodebase` flag? If so - is this comment needed here?